react-web/src/pages/certification/Certification.tsx

```diff
@@ -248,7 +249,11 @@ const Certification = () => {
 
   return (
     <>
-      <div
+      {!subscribedFeatures?.length || subscribedFeatures?.indexOf("l1-run") === -1 ? 
+      <div id="searchContainerEmpty">
+        <p>You do not have enough subscriptions to perform the test run. <br/> <Link to="/subscription">Click here</Link> to review our packages and pick one to proceed.</p>
```

Comment:
my English is far from being my strong point, but this sounds a bit strange to me. 
How many subscriptions do I need to perform a test? :) 
Shouldn't be more related to something like: you don't have a valid subscription to perform bla bla bla?

Reply:
Done